driver-core/src/test/functional/com/mongodb/operation/FindOperationSpecification.groovy

```diff
@@ -153,6 +156,22 @@ class FindOperationSpecification extends OperationFunctionalSpecification {
         async << [true, false]
     }
 
+    def 'should query with exhaust set'() {
```

Comment:
Is this actually testing anything?  The only thing that really needs to be tested is that the exhaust flag is passed from FindOperation to QueryBatchCursor, and I'm not sure how to actually test that..

Reply:
I think you're right. It's also a bit repetitive as well, so i've removed it!